File: frontend/app/_utils/firebase.js
// Import the functions you need from the SDKs you need
import { initializeApp } from "firebase/app";
import {getAuth} from 'firebase/auth'
import { getFirestore } from 'firebase/firestore';
// TODO: Add SDKs for Firebase products that you want to use
// https://firebase.google.com/docs/web/setup#available-libraries

// Your web app's Firebase configuration
// For Firebase JS SDK v7.20.0 and later, measurementId is optional
const firebaseConfig = {
  apiKey: "AIzaSyA4Krn4cuEumQHVVS1oRxaaAkHm2h7Suw8",
  authDomain: "hi7-hl7.firebaseapp.com",
  projectId: "hi7-hl7",
  storageBucket: "hi7-hl7.appspot.com",
  messagingSenderId: "410837804656",
  appId: "1:410837804656:web:38fe4aeaccb29d1eee36b5",
//   measurementId: "G-FKBDD0CQC8"
};

// Initialize Firebase
const app = initializeApp(firebaseConfig);
export const auth = getAuth(app);
export const db = getFirestore(app);

File: database-default/firebase.py
import pyrebase

config = {
  "apiKey": "AIzaSyA4Krn4cuEumQHVVS1oRxaaAkHm2h7Suw8",
  "authDomain": "hi7-hl7.firebaseapp.com",
  "databaseURL": "https://hi7-hl7-default-rtdb.firebaseio.com",
  "projectId": "hi7-hl7",
  "storageBucket": "hi7-hl7.appspot.com",
  "messagingSenderId": "410837804656",
  "appId": "1:410837804656:web:38fe4aeaccb29d1eee36b5",
  "measurementId": "G-WK8LCJP128"
}

firsebase = pyrebase.initialize_app(config)
database = firebase.database()

data = {"Name": "Hola", 
        "AHC/WCB#": 00000000, 
        "Address": "SAIT NW b", 
        "Date of Birth": "2000/01/01", 
        "Phone Number": "123-456-7899", 
        "Referring Physician": "MIKE"}

database.push(data)
